.github/workflows/build-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Build test images
+on:
+#   push:
+#     branches:
+#       - "main"
+  pull_request: {}
+
+env:
+  IMAGE_NAME_E2E: crc-e2e
+  IMAGE_NAME_INTEGRATION: crc-integration
+
+jobs:
+  build-e2e:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      
+      - name: Build e2e image
+        env:
+            PR_NUMBER: ${{ github.event.number }}
+        run: |
+          CRC_E2E_IMG_VERSION=pr-${PR_NUMBER} make containerized_e2e
+      
+      - name: Archive e2e image
+        env:
+            PR_NUMBER: ${{ github.event.number }}
+        run: |
+          podman save -o ${{ env.IMAGE_NAME_E2E }}.tar quay.io/crcont/${{ env.IMAGE_NAME_E2E}}:pr-${PR_NUMBER}
+
+      - name: Upload e2e artifact
```

Comment:
Would change to Upload e2e image (to match previous naming convention for steps)